src/liboslexec/llvm_instance.cpp

```diff
@@ -1294,8 +1294,13 @@ BackendLLVM::initialize_llvm_group()
         TypeSpec rettype = OSLCompilerImpl::type_from_code(types, &advance);
         types += advance;
         std::vector<llvm::Type*> params;
+        // bool debug = OIIO::Strutil::contains(funcname, "range");
```

Comment:
Did you mean to leave these commented out lines here?

Reply:
no, that's a mistake, will remove